ui_test/Cargo.toml

```diff
@@ -18,4 +18,4 @@ lazy_static = "1.4.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 color-eyre = { version = "0.6.1", default-features = false, features = ["capture-spantrace"] }
-
+cargo_metadata = "0.15"
```

Comment:
Hm, cargo-miri calls `cargo metadata` and parses the JSON via serde itself... any idea why it avoids depending on cargo_metadata? How much slower does this make the build?

Reply:
not in any relevant way, the two new dependencies (`cargo-metadata` and `camino`) build in under a second and `camino` will go away with the next `cargo-metadata` version bump

Reply:
Nice, then we should probably also use it for cargo-miri. (Not in this PR though.)
I think what happened is that in https://github.com/rust-lang/miri/pull/1540 I removed the dependency, and then recently @saethlin added back a use of the `cargo metadata` command for local crate detection, but without using the cargo-metadata crate.